tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -7,10 +7,6 @@ summary: Learn about how to audit a cluster in TiDB Cloud.
 
 TiDB Cloud provides you with a database audit logging feature to record a history of user access details (such as any SQL statements executed) in logs.
 
```

Comment:
@ljun0712  Since this feature is GA now, please also add the link of this doc to the `TOC-tidb-cloud.md` file so users can find this doc in the document left navigation pane. Thanks.